package.json

```diff
@@ -94,11 +94,13 @@
     "issue-parser": "7.0.1",
     "lodash.get": "4.4.2",
     "lodash.merge": "4.6.2",
+    "make-synchronized": "0.6.0",
```

Comment:
This isn't used?

Reply:
Sorry my bad, that is try to make load config to synchronized, but that have some problems

I'm already drop that at d3e7fbb